src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -111,6 +114,42 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
                 .orElse(personToEdit.getMatriculationNumber());
         CovidStatus updatedCovidStatus = editPersonDescriptor.getCovidStatus()
                 .orElse(personToEdit.getStatus());
+
+        if (editPersonDescriptor.getName().isPresent()) {
```

Comment:
In my opinion, maybe this part can be abstracted into separate method so that it will be easier to understand and will not be too long.

Reply:
👍  I agree with Jia Le's comment

Reply:
Thank you @DannyDakota and @jiale-c ! I have abstracted it into a different method 👍